handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -120,23 +128,35 @@ func newTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
 	}, nil
 }
 
-func claimAsStrings(t *jwt.Token, claim string) ([]string, bool) {
+func NewTokenInfo(t *jwt.Token, timeBase time.Time) (*TokenInfo, error) {
+	issuer, ok := ClaimAsString(t, JwtClaimIssuer)
+	if ok {
+		processor, found := options.AppSettings.JwtProcessors[issuer]
+		if found {
+			log.Printf("token matched processor for issuer %s", issuer)
+			return processor.(JwtProcessor).Process(t, timeBase)
+		}
+	}
+	return defaultNewTokenInfo(t, timeBase)
+}
+
+func ClaimAsStrings(t *jwt.Token, claim string) ([]string, bool) {
```

Comment:
You don't need to export ClaimAsStrings() to other packages, so use lowercase claimAsStrings() instead.